src/main/java/org/gridsuite/network/map/model/TapChangerData.java

```diff
@@ -17,6 +19,44 @@
 @Getter
 @EqualsAndHashCode
 public class TapChangerData {
-    Integer lowTap;
-    Integer highTap;
+    private Integer lowTap;
+
+    private Integer tapPosition;
+
+    private Integer highTap;
```

Comment:
`highTapPosition` ? Seems you don't need it in other services by the way...

Reply:
✓